src/index.js

```diff
@@ -54,21 +63,14 @@ function _smoothScroll({ scrollTo, offset, duration, container, updateHistory, h
 
 const VueSmoothScroll = {
   install(Vue, config) {
-    const defaultValue = {
-      duration: 500,
-      offset: 0,
-      container: window,
-      updateHistory: true,
-    };
-
     Vue.directive('smooth-scroll', {
       inserted(el, binding, vnode) {
         // Do not initialize smoothScroll when running server side, handle it in client
         // We do not want this script to be applied in browsers that do not support those
         // That means no smoothscroll on IE9 and below.
         if (typeof window !== 'object' || window.pageYOffset === undefined) return;
 
-        let resolvedConfig = Object.assign({}, defaultValue);
+        let resolvedConfig = Object.assign({}, getDefaultConfig());
```

Comment:
Could've been simplified to `resolvedConfig = getDefaultConfig()` as there's no possibility of modifying the default config anymore. And line 110 below to `Object.assign(getDefaultConfig(), ..)`